mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraACLMapper.java

```diff
@@ -66,11 +67,14 @@
     private final PreparedStatement conditionalUpdateStatement;
     private final PreparedStatement readStatement;
     private final PreparedStatement deleteStatement;
+    private final ConsistencyLevel consistencyLevel;
 
     @Inject
-    public CassandraACLMapper(Session session, CassandraUserMailboxRightsDAO userMailboxRightsDAO, CassandraConfiguration cassandraConfiguration) {
+    public CassandraACLMapper(Session session, CassandraUserMailboxRightsDAO userMailboxRightsDAO,
+                              CassandraConfiguration cassandraConfiguration, CassandraConsistenciesConfiguration consistenciesConfiguration) {
```

Comment:
Why isn't CassandraConsistenciesConfiguration part of CassandraConfiguration ? It sounds weird to pass two configuration objects...

Reply:
`CassandraConfiguration` does not parse, it holds a bunch of `int`, checking they are positive.

Here we want plain `ConsistencyLevel`s.

Moreover, it respect the ISP, which is always nice.